tr/console-and-shells.rst

```diff
@@ -485,7 +489,7 @@ Kanca metodları
 Seçenekleri Yapılandırmak ve Yardım Üretmek
 ===========================================
 
-.. php:class:: ConsoleOptionParser
+.. php:class OptionParser
```

Comment:
are you sure those changes are correct?

Reply:
Yea, that was caused by me.  I missed reverting a couple places when squashing earlier.  Not fully verifying every regex change can be dangerous!

Reply:
Hmm, no, sorry, I don't know what this specific change is.  I'm not sure how I would have generated that.  sigh.

Reply:
I know what it is now. I had originally marked them with .. code-block:: console but changed it to .. code-block:: bash.

I won't be making large changes like that again :)